just_audio_background/lib/just_audio_background.dart

```diff
@@ -238,9 +238,8 @@ class _JustAudioPlayer extends AudioPlayerPlatform {
       playerDataController.stream;
 
   @override
-  Future<LoadResponse> load(LoadRequest request) async {
-    return _playerAudioHandler.customLoad(request);
-  }
+  Future<LoadResponse> load(LoadRequest request) =>
+      _playerAudioHandler.customLoad(request);
 
   @override
   Future<PlayResponse> play(PlayRequest request) async {
```

Comment:
This async can be deleted.

Reply:
There's `await _audioHandler.play();` next line, so It's not really possible to remove the `async` except for using `.then`?

Reply:
Sorry, I was again mixed up with the GitHub viewing experience. This is correct as is.